crates/biome_css_analyze/tests/specs/nursery/noInvalidPositionAtImportRule/validCharset.css

```diff
@@ -0,0 +1,2 @@
+@charset 'utf-8';
+@import 'foo.css';
```

Comment:
I suppose `@charset 'utf-8'; @import 'foo.css';` passes as well?

Reply:
Thank you for your review.
fix: https://github.com/biomejs/biome/pull/2717/commits/eb6210b69fd39afd6435df3db64282691050ad54